rudder-server-relay/SPECS/rudder-server-relay.spec

```diff
@@ -320,6 +320,7 @@ fi
 # Add required includes in the apache2 configuration
 if ! grep -qE "^. /etc/sysconfig/rudder-relay-apache$" /etc/sysconfig/apache2; then
```

Comment:
this will not fix existing installs, shouldn't be a separate check for the variable?